cost/azure/reserved_instances/mca_recommendations/README.md

```diff
@@ -1,15 +1,19 @@
-# Azure MCA Reserved Instances Recommendation Policy Template
+# Azure Reserved Instances Recommendation
+
+## NOTE:  This policy supersedes the previous [Policy Template](../recommendations/) that used the Azure EA Key
 
 ## What it does
 
-This Policy Template leverages the [Azure API](https://docs.microsoft.com/en-us/rest/api/consumption/reservationrecommendations/list). It will raise incidents if Azure MCA has any RI Purchase Recommendations, whose net savings exceeds the `Net Savings Threshold` parameter in the Policy.
+This Policy leverages the [Azure API](https://docs.microsoft.com/en-us/rest/api/consumption/reservationrecommendations/list). It will raise incidents if Azure has any Reservations Purchase Recommendations, whose net savings exceeds the `Net Savings Threshold` parameter in the Policy.  Supports Shared subscription Reservations and the following Reservation Types: ['VirtualMachines', 'SQLDatabases', 'PostgreSQL', 'ManagedDisk', 'MySQL', 'RedHat', 'MariaDB', 'RedisCache', 'CosmosDB', 'SqlDataWarehouse', 'SUSELinux', 'AppService', 'BlockBlob', 'AzureDataExplorer', 'VMwareCloudSimple'].  You must use the Resource Type input to apply the policy for each resource type you want a reservation recommendation.
```

Comment:
```suggestion
This Policy leverages the [Azure API](https://docs.microsoft.com/en-us/rest/api/consumption/reservationrecommendations/list). It will raise incidents if Azure has any Reservations Purchase Recommendations, whose net savings exceeds the `Net Savings Threshold` parameter in the Policy.  Supports Shared subscription Reservations and the following Reservation Types: ['VirtualMachines', 'SQLDatabases', 'PostgreSQL', 'ManagedDisk', 'MySQL', 'RedHat', 'MariaDB', 'RedisCache', 'CosmosDB', 'SqlDataWarehouse', 'SUSELinux', 'AppService', 'BlockBlob', 'AzureDataExplorer', 'VMwareCloudSimple'].  You must use the Resource Type input to apply the policy for each resource type you want a reservation recommendation.
```